seed/models/meters.py

```diff
@@ -64,7 +64,7 @@ class Meter(models.Model):
         (ELECTRICITY_UNKNOWN, 'Electric - Unknown'),
     )
 
-    type_lookup = dict((reversed(type) for type in ENERGY_TYPES))
+    type_lookup = dict((reversed(type) for type in ENERGY_TYPES))  # type: ignore
```

Comment:
Great example of types not working well sometimes. mypy is unable to infer that the Tree has a decorator which evaluates the tree (in this case into a float)